src/controllers/terminal.go

```diff
@@ -0,0 +1,30 @@
+package controllers
+
+import (
+	"fmt"
+	"github.com/dana-team/platform-backend/src/terminal"
+	"github.com/dana-team/platform-backend/src/types"
+	"go.uber.org/zap"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/remotecommand"
+)
+
+// HandleStartTerminal Handles execute shell API call
+func HandleStartTerminal(clientSet kubernetes.Interface, config *rest.Config, clusterName, namespaceName, podName, containerName, shell string, logger *zap.Logger) (types.StartTerminalResponse, error) {
+
+	sessionID, err := terminal.GenTerminalSessionId()
+	if err != nil {
+		logger.Error(fmt.Sprintf("coundn't generate terminal_utils session for pod %s and container %s in namespace %s on cluster %s with err: %s",
```

Comment:
Most/all of our errors in the `controllers` package are in the format of an error constant, i.e.

https://github.com/dana-team/platform-backend/blob/8f0f6a65689576bf566e7f7f922b4e38b7200453/src/controllers/capp.go#L33-L34